test/unit/client.js

```diff
@@ -328,6 +328,48 @@ describe('Detect node environment', () => {
 	});
 });
 
+describe('Client deleteUsers', () => {
+	it('should allow completely optional options', async () => {
+		const client = await getClientWithUser();
+
+		client.post = () => Promise.resolve();
+
+		await expect(client.deleteUsers(['_'])).to.eventually.equal();
+	});
+
+	it('delete types - options.conversations', async () => {
+		const client = await getClientWithUser();
+
+		client.post = () => Promise.resolve();
+
+		await expect(client.deleteUsers(['_'], { conversations: 'hard' })).to.eventually.equal();
+		await expect(client.deleteUsers(['_'], { conversations: 'soft' })).to.eventually.equal();
```

Comment:
Shouldn't we test for option `'pruning'` too?

Reply:
That is not supported by `conversations` option but I can add it as an extra case just in case? 🤔